packages/atomic/src/pages/examples/ipx.html

```diff
@@ -63,12 +72,57 @@
         width: 5rem;
         height: 1.25rem;
       }
+
+      atomic-recs-list::part(label) {
+        font-size: var(--atomic-text-xl);
+        padding-top: 0.75rem;
+        padding-bottom: 0.25rem;
+      }
+
+      atomic-recs-list::part(outline) {
+        padding: 0.5rem;
+        border: none;
+      }
+
+      atomic-recs-list::part(result-list-grid-clickable) {
+        inset: unset;
+      }
+
+      atomic-recs-list::part(result-list-grid-content) {
+        width: 27rem;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
```

Comment:
I prefer to keep result styling scoped to specific templates. See [displaying results (styling)](https://docs.coveo.com/en/atomic/latest/usage/displaying-results/#styling).